src/Microsoft.DocAsCode.Build.Engine/HostService.cs

```diff
@@ -623,6 +655,7 @@ internal enum LoadPhase
     {
         None,
         PreBuild,
-        PostBuild
+        PostBuild,
+        PostPostBuild,
```

Comment:
requesting for better name:)

---
In reply to: [77579915](https://github.com/dotnet/docfx/pull/670#discussion_r77579915) [](ancestors = 77579915)